SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/EventTracingTests/EventTracingTest.h

```diff
@@ -29,6 +29,11 @@ class SPATIALGDKFUNCTIONALTESTS_API AEventTracingTest : public ASpatialFunctiona
 
 protected:
 	FName ReceiveOpEventName = "worker.receive_op";
+	FName SendPropertyUpdatesEventName = "unreal_gdk.send_property_updates";
```

Comment:
I think we could probably test these a bit more thoroughly if we collected all the spans and checked the cause tree had the expected spans in the history but that would be a follow up task for sure if you think that would be relevant/important

Reply:
I think it's defo a good test. Will pop something in the backlog.